ibis/sql/tests/test_sqlalchemy.py

```diff
@@ -104,6 +104,32 @@ def test_sqla_schema_conversion(self):
 
         assert_equal(schema, expected)
 
+    def test_sqla_postgres_schema_conversion(self):
```

Comment:
@ian-r-rose as you are using a specific dialect here I think it would be good to move this test to https://github.com/ibis-project/ibis/blob/master/ibis/sql/postgres/tests/test_client.py

thanks @ian-r-rose  for working on that. Looks very nice! after this change we can merge this PR!

Reply:
Okay, should be set!